openpype/hosts/houdini/plugins/publish/validate_review_colorspace.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import PublishValidationError
+from openpype.pipeline.publish import RepairAction
+from openpype.hosts.houdini.api.action import SelectROPAction
+
+import os
+import hou
+
+
+class SetDefaultViewSpaceAction(RepairAction):
+    label = "Set default view space"
+    icon = "mdi.monitor"
+
+
+class ValidateReviewColorspace(pyblish.api.InstancePlugin):
+    """Validate Review Colorspace parameters.
+
+    It checks if 'OCIO Colorspace' parameter was set to valid value.
+    """
+
+    order = pyblish.api.ValidatorOrder + 0.1
+    families = ["review"]
+    hosts = ["houdini"]
+    label = "Validate Review Colorspace"
+    actions = [SetDefaultViewSpaceAction, SelectROPAction]
+
+    def process(self, instance):
+        invalid = self.get_invalid(instance)
+        if invalid:
+            raise PublishValidationError(
+                ("'OCIO Colorspace' parameter is not valid."),
+                title=self.label
+            )
+
+    @classmethod
+    def get_invalid(cls, instance):
+
+        rop_node = hou.node(instance.data["instance_node"])
+        if os.getenv("OCIO") is None:
+            cls.log.warning(
+                "Default Houdini colorspace is used, "
+                " skipping check.."
+            )
```

Comment:
Not sure if this should be a warning. It's totally fine for a production to be not color managed and if it is, it's not really a point to warn the artists about - that's more of an "admin problem" maybe? Debug log might be better suited so it doesn't show in artist facing reports.